robotiq_s_model_articulated_msgs/msg/SModelRobotInput.msg

```diff
@@ -0,0 +1,154 @@
+# gACT : Initialization status, echo of the rACT bit (activation bit).
```

Comment:
It's best practice for messages to be in their own package in order to prevent circular dependencies.  Deleting the `robotiq_s_moedel_control` message will break things.  We generally like to deprecate in one version and then remove in the next.